docs/manual/interpreterinstallation.md

```diff
@@ -140,6 +140,11 @@ You can also find the below community managed interpreter list in `conf/interpre
     <td>HTML and AngularJS view rendering</td>
   </tr>
   <tr>
+    <td>beam</td>
+    <td>org.apache.zeppelin:zeppelin-beam:0.6.1</td>
+    <td>Beam interpreter</td>
+  </tr>
+  <tr>
```

Comment:
This change seems unnecessary since we don't support zeppelin-beam in zeppelin 0.6.1.